### ISSUE TITLE
remove 🔍 and allow click directly on table row to show details

### ISSUE THREAD

**Issue by emanuele-em** [emanuele-em/man-in-the-middle-proxy#24]
It should be better and more ux prone to remove the 🔍 button and allow click directly on table row to show details

**Comment by sak96** [comment]
pending on: emilk/egui/issues/2761

**Comment by sak96** [comment]
do you think using tauri and yew seems feasible ?
reasoning: Web UI are very much extensible when compared to egui.

**Comment by emanuele-em** [comment]
yes you are totally right, I choose egui only to learn something about native gui, in this case web ui should be 3 times better

**Comment by emanuele-em** [comment]
@sak96 if you have time we can try to migrate to tauri + sveltekit / vue / react

**Comment by emanuele-em** [comment]
yes, why not! I never used it but it's ok
